src/test/java/com/suse/saltstack/netapi/utils/SaltStackClientTest.java

```diff
@@ -0,0 +1,65 @@
+package com.suse.saltstack.netapi.client;
```

Comment:
This package does not match the file system path above, please fix the path in this case.